### PR TITLE
Fix missing templates

### PR DESCRIPTION
**Pull request by tu-nv**
### Problem
In my Open edX Hawthorn, when installing xblock-submit-and-compare, template files are not copied. 
### Solution
Add _templates_ to _package_data_.

### CODE REVIEW

**Comment by caesar2164** [comment]
Thanks for the PR @tu-nv!

I'll be looking at it this sprint (next two weeks), sorry for the delay.

**Comment by caesar2164** [comment]
Hi @tu-nv,

I tried running your branch in a docker Hawthorn devstack and I get:
screen shot 2018-11-29 at 11 35 09

Could you see if you can reproduce & fix it?
(Copying this PR should help: https://github.com/Stanford-Online/xblock-free-text-response/pull/35)

File: submit_and_compare/submit_and_compare.py [hunk 1]
```diff
@@ -245,9 +247,9 @@ def student_view(self, context=None):
                 'is_past_due': self.is_past_due(),
             }
         )
-        template = get_template('submit_and_compare_view.html')
```
Comment (tu-nv) [inline submit_and_compare/submit_and_compare.py:248]: I am not sure why get_template does not work. Seem to depend on the change on the Django side.

**Comment by caesar2164** [comment]
Hi @tu-nv, @OmarIthawi already made a similar fix in his PR: https://github.com/Stanford-Online/xblock-free-text-response/pull/35.

I think you should be able to crib some (if not all) of his changes to `get_template`.

**Comment by tu-nv** [comment]
Hi @caesar2164 I already fix the template error in https://github.com/Stanford-Online/xblock-submit-and-compare/pull/25/commits/cedf5e5ff2e145838a08327a4f90167a1559a570 
@OmarIthawi and I use the same method: replacing get_template by resource_loader. The only problem is that I am not clear why get_template does not work for xblock in Hawthorn.

**Comment by OmarIthawi** [comment]
@tu-nv I don't have it on top of my head now, but I know that `loader.render_django_template` is much better than `get_template` since the former supports `i18n_service` which allows translating the templates.

This reason is enough for me use `loader.render_django_template` over `get_template`.

**Comment by caesar2164** [comment]
@tu-nv sorry for the lack of response. I'm spinning up your most recent changes and I'll get back to you with a few more comments in the next few days.

**Review by caesar2164 — APPROVED** [review]
This works for me now! Thanks for your patience, @tu-nv.

Could you squash your changes?

**Review by stvstnfrd — APPROVED** [review]

**Comment by caesar2164** [comment]
@tu-nv please push the version to `0.7.0` and squash everything together. (see this commit: https://github.com/Stanford-Online/xblock-submit-and-compare/commit/a0f0900403c360a0e4c39cb0730ec442ce8b5e96)

thanks!

**Comment by stvstnfrd** [comment]
@caesar2164 Have you PR'ed to bump in platform?

**Comment by caesar2164** [comment]
Not yet.

**Pull request merged**